packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -108,12 +114,13 @@ const FormLabel = ({
   }
 
   const Component = as;
+  const isLabelLeftPositioned = position === 'left';
   // only set 120px label when device is desktop
-  const width = position === 'left' && isDesktop ? '120px' : 'auto';
+  const width = isLabelLeftPositioned && isDesktop ? '120px' : 'auto';
```

Comment:
Optional: I think we should move this width to somewhere on the top as a `SOME_CONSTANT` for ease of readability.

Reply:
We should have a `formTokens.js` where we can store this. Doing the same for other components where with fixed dimensions

Reply:
it's just one tiny thing don't think that creating a whole file for just this will add any value.